compiler/ballerina-lang/src/main/java/io/ballerina/projects/DependencyGraph.java

```diff
@@ -84,6 +84,10 @@ public void findCycles() {
         return topologicallySortedNodes;
     }
 
+    Map<T, Set<T>> dependencies() {
```

Comment:
I have a concern about exposing the internal data structure of the dependency graph.  

We can refactor this later.